aviary/utils/set_mass_defaults.py

```diff
@@ -2,6 +2,9 @@
 
 
 def mass_defaults(prob):
+    """
+    This function is not used. Should we keep this file?
+    """
     prob.model.set_input_defaults(
```

Comment:
I suspect some of these made their way into the paramport.  If it isn't used, we can safely delete. (We can always get it back from the commit history, if needed.)

Reply:
I removed `set_mass_defaults.py` now.